cypress/e2e/0-JoinBet/footer/QA213__Patr1.cy.js

```diff
@@ -0,0 +1,101 @@
+//Hi, guys!
+
+describe('Part 1 of Test QA213',()=>{
+    it('Column 1',()=>{
```

Comment:
@ElKalinin предлагаю сразу называть describe и it логически из того что тестируем, например:

```suggestion
describe('Casino. Footer', () => {
    it('Отображение всех текстовых данных - язык ru', () => {
```